lib/clearwater/component.rb

```diff
@@ -27,12 +27,8 @@ def tag tag_name, attributes=nil, content=nil
     end
 
     def to_s
-      html = render.to_s
-      if html.respond_to? :html_safe
-        html = html.html_safe
-      end
-
-      html
+      html = Array(render).map(&:to_s).join
```

Comment:
`Array#join` calls `to_s` on all elements of the array. :-) This can be simplified to `Array(render).join`.